api/package.json

```diff
@@ -14,7 +14,7 @@
     "seed": "knex seed:run",
     "seed-dev": "NODE_ENV=development DEV_DB_HOST=localhost npm run seed",
     "start": "node main.js",
-    "start-dev": "NODE_ENV=development DEV_DB_HOST=localhost nodemon ./main.js -e js",
```

Comment:
nodemon looks at the api codebase, and restarts when it sees changes. Useful for api development.

`DEV_DB_HOST` is set to localhost, when developing locally. I added this, because I'm using a local db to develop. It's ok if we remove this. I added it because it was helpful for me. But, if it's not helpful for everyone, then it should go.